src/components/MyAccountContent/MyAccountContent.js

```diff
@@ -1,9 +1,25 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+import { useSelector } from 'react-redux';
 import { WrapStyled } from './MyAccountContentStyled';
+import GracePeriodError from '../GracePeriodError/GracePeriodError';
 
-const MyAccountContent = ({ children }) => <WrapStyled>{children}</WrapStyled>;
+const MyAccountContent = ({ children }) => {
+  const currentPlan = useSelector(state => state.planDetails.currentPlan);
+
+  const isPeriodError = currentPlan.some(
+    ({ status, expiresAt }) =>
+      status === 'active' && new Date(expiresAt) < new Date()
+  );
+
+  return (
+    <WrapStyled>
+      {isPeriodError && <GracePeriodError />}
+      {children}
+    </WrapStyled>
+  );
+};
```

Comment:
we need the same logic (to display GracePeriodError) in PlanDetails component and probably more (to discuss)